client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -274,18 +277,9 @@ private String createBasicAuthString() {
                     return false;
                 }
                 if (!(throwable instanceof WebException)) {
-                    // Don't retry posts when a NoSuchElementException is raised since the request might have ended up at the server the first time
-                    if (throwable instanceof NoSuchElementException && POST.equals(fullReq.getHttpMethod())) {
-                        return false;
-                    }
-
                     // Retry on system error of some kind, like IO
                     return true;
```

Comment:
This was the row that triggered the retry before. It was designed this way, without early exit for POST, in order to retry POSTs if the reason is a local error, like a DNS lookup failure or something that means that the request has not left this system. I guess this is now more up to the caller, to add approriate retries for POSTs.

Reply:
Do you think its better to "blacklist" exceptions for post call retries, so we just add the ReadTimeOutException in the condtion?

Reply:
No, I think you did the right thing here. Since we have had problems with requests being retried, rather than not being retried, I think this is the way to go. Blacklisting exceptions is hard here, there could be other low level exceptions that are hard to know if they ment the request was sent or not. So my comment was more of a clarification.